dap-launch.el

```diff
@@ -29,8 +29,10 @@
 
 (defun dap-launch-find-launch-json ()
   "Return the location of the launch.json file in the current project."
-  (when-let ((project (lsp-workspace-root)))
-    (concat project "/launch.json")))
+  (when-let ((project (lsp-workspace-root))
```

Comment:
I think this should be:
```suggestion
  (-when-let* ((project (lsp-workspace-root))
```

Reply:
No. when-let allows that. If you dig trough its source, you will find a reference to if-let*, being defined as (let*).